pycytominer/cyto_utils/DeepProfiler_processing.py

```diff
@@ -256,3 +309,154 @@ def aggregate_deep(self):
 
         df_out = self.aggregated_profiles
         return df_out
+
+
+class SingleCellDeepProfiler:
+
+    """This class holds functions needed to analyze single cells from the DeepProfiler (DP) run. Only pycytominer.normalization() is implemented.
+
+    Attributes
+    ----------
+    deep_data : DeepProfilerData
+        DeepProfilerData object to load data from DeepProfiler project
+    aggregated_profiles : pandas.DataFrame
+        df to hold the metadata and profiles.
+    file_aggregate : dict
+        dict that holds the file names and metadata.
+        Is used to load in the npz files in the correct order and grouping.
+    output_file : str
+        If provided, will write annotated profiles to folder. Defaults to "none".
+
+    Methods
+    -------
+    normalize(profiles, features, image_features, meta_features, samples, method, output_file, compression_options,
+    float_format, mad_robustize_epsilon, spherize_center, spherize_method, spherize_epsilon)
+        normalize profiling features from DeepProfiler run with pycytominer.normalize()
+
+    Example
+    -------
+    import pathlib
+    from pycytominer.cyto_utils import DeepProfiler_processing
+
+    index_file = pathlib.Path("path/to/index.csv")
+    profile_dir = pathlib.Path("path/to/features/")
+
+    deep_data = DeepProfiler_processing.DeepProfilerData(index_file, profile_dir, filename_delimiter="/", file_extension=".npz")
+    deep_single_cell = DeepProfiler_processing.SingleCellDeepProfiler(deep_data)
+    normalized = deep_single_cell.normalize_deep_single_cells()
+    """
+
+    def __init__(
+        self,
+        deep_data: DeepProfilerData,
+    ):
+        """
+        __init__ function for this class.
+
+        Arguments
+        ---------
+        See above for all parameters.
+        """
+
+        self.deep_data = deep_data
+
+    def setup_normalize(self):
+        """
+        Sets up the single_cells attribute. This is a helper function to normalize_deep_single_cells().
+
+        single_cells is a pandas dataframe in the format expected by pycytominer.normalize()
+        """
```

Comment:
```suggestion
    def get_singlecells(self, output=False):
        """
        Sets up the single_cells attribute or output as a variable. This is a helper function to normalize_deep_single_cells().
        single_cells is a pandas dataframe in the format expected by pycytominer.normalize().
        
        Arguments
        -----------
        output : bool
            If true, will output the single cell dataframe instead of setting to self attribute
        """
```